lib/account/pages/profile_page.dart

```diff
@@ -330,6 +309,12 @@ class _ProfilePageState extends State<ProfilePage> {
     );
   }
 
+  Future<void> _pushAndLoadProfile(Widget page) async {
```

Comment:
the name is confusing - lt sounds like it is loading the profile immediately. Maybe "_pushEditPage", the exact implementation does not need to be exposed.